planet/planet.go

```diff
@@ -461,3 +468,11 @@ func injectAssetIntoMetadata(feature *geojson.Feature, asset Asset) {
 		feature.Properties["type"] = asset.Type
 	}
 }
+
+func EvalDateParam(context *Context, date string, field string) (eval bool, err error) {
+	if eval, _ := regexp.MatchString("^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]+)?Z", date); !eval {
```

Comment:
Note that I believe Planet Labs uses RFC3339 for its dates. If that is correct, you can use time.RFC3339 as its parse string.